helm/flux-app/templates/install.yaml

```diff
@@ -902,7 +927,12 @@ spec:
               path: /
               port: http
           resources:
-{{ include "resources.sourceController" . | indent 12 }}
```

Comment:
I am not sure about these, I assume the last time these were added to the kustomizations then kuztomize was not rerun.

Reply:
It was a VPA fix by @mproffitt https://github.com/giantswarm/flux-app/pull/151/files